Sources/KIF/Additions/UIView-KIFAdditions.m

```diff
@@ -1062,4 +1099,22 @@ -(UIView*)tryGetiOS16KeyboardFallbackViewFromParentView:(UIView*) parentView {
     return nil;
 }
 
+- (nullable UIView *)parentPlainScrollView
+{
+    // We don't want collection view and table view because we handle them separately.
+    // This function is only getting a plain scroll view
+    UIView *currentSuperview = self.superview;
```

Comment:
What about the scenario where the element being tapped on is a descendant rather than a direct subview of a scroll view? In more complex controls, I'd think this would happen relatively often.

Reply:
Trying to picture this in my head, can you provide an example?

Reply:
IF it's a `UIScrollview` that contains a `UIView` that contains a `UILabel` for instance`

So something visually like:

```
> UIScrollView
--> UIView
---->UILabel ** This one
--> UITextView
```

Reply:
Yep, something like that is what I was thinking

Reply:
@justinseanmartin I think I am already doing that for the `while loop` no?

Reply:
Yep, you're right, sorry I misunderstood. Would recommend using the name `ancestor` here to make it a bit more clear.